src/status_im2/contexts/syncing/device/view.cljs

```diff
@@ -0,0 +1,31 @@
+(ns status-im2.contexts.syncing.device.view
+  [:require
+   [quo2.core :as quo]
+   [utils.i18n :as i18n]
+   [status-im2.contexts.syncing.device.style :as style]])
+
+(defn view
+  [{:keys [name
+           this-device?
+           device-type
+           enabled?]}]
+  (let [paired?   (and (not this-device?) enabled?)
+        unpaired? (not enabled?)]
+    [quo/settings-list
+     (merge
+      {:container-style style/device-container
+       :title           name
+       :override-theme  :dark
+       :left-icon       (if (= device-type :mobile) :i/mobile :i/desktop)}
+      (when unpaired?
+        {:button-props {:title    (i18n/label :t/pair)
+                        :on-press #(js/alert "feature not added yet")}})
+      (when paired?
+        {:button-props {:title    (i18n/label :t/unpair)
+                        :on-press #(js/alert "feature not added yet")}})
+      (when this-device?
+        {:status-tag-props {:size           :small
+                            :status         {:type :positive}
+                            :no-icon?       true
+                            :label          (i18n/label :t/this-device)
+                            :override-theme :dark}}))]))
```

Comment:
I think it'd be better to change this for `cond->` because this is exactly the use case for cond-> and also using `assoc` instead  of `merge` is cheaper.

To be clear, I'm suggesting the following pattern:
```clojure
(cond-> {:your :map}
  bool-val? (assoc :another-key "another value")
)

```